Add ADP-specific glossary terms for GA (#1691)
redpanda-data/documentation: modules/terms/partials/governance-dashboard.adoc

> === Governance Dashboard

Author: ```suggestion
=== Governance dashboard
```

> === Governance dashboard

Author: ```suggestion
=== governance dashboard
```

> :term-name: Governance Dashboard

Author: ```suggestion
:term-name: Governance dashboard
```

> :term-name: Governance dashboard

Author: ```suggestion
:term-name: governance dashboard
```

> :hover-text: Cross-tenant overview UI showing spending, agent activity, MCP server inventory, and authorization events. V0 (GA) composes data from `SpendingService`, `AgentRegistryService`, and `MCPServerService`.

Author: ```suggestion
:hover-text: Cross-tenant overview UI showing spending, agent activity, MCP server inventory, and authorization events. The dashboard composes data from `SpendingService`, `AgentRegistryService`, and `MCPServerService`.
```
